eta/core/datasets.py

```diff
@@ -1847,11 +1880,9 @@ def build(self, path, description=None, pretty_print=False,
                 data_method=data_method
             )
 
-            # MOVE here is used to rename the file
-            dataset.add_file(data_path, labels_path,
-                             os.path.basename(record.new_data_path),
-                             os.path.basename(record.new_labels_path),
-                             file_method=MOVE)
+            # The `file_method` is irrelevant because the files are already in
+            # the dataset directory
+            dataset.add_file(data_path, labels_path)
```

Comment:
Is this because the copy is performed in the previous line? (`record.build`)

Reply:
Correct.  If the file already exists in the dataset directory, then `dataset.add_file()` doesn't do anything to files, it just adds them to the manifest.